### ISSUE TITLE
Error reading config: open .bazel_fix_commands.json: no such file or directory

### ISSUE THREAD

**Issue by flolu** [bazelbuild/bazel-watcher#369]
I have a [ts_devserver](https://github.com/flolu/cents-ideas/blob/b90abb1ac40dfe811f716b56799cf7b5fca5036f/services/client/src/BUILD.bazel#L69) in one of my BUILD files which serves an Angular application. Everything works fine, but in the console there is always an error:
```
iBazel [8:10PM]: Error reading config: open .bazel_fix_commands.json: no such file or directory
```
![Screenshot from 2020-03-26 20-18-48](https://user-images.githubusercontent.com/34191949/77687712-233e7380-6f9f-11ea-96c2-6314416b4da1.png)

`.bazel_fix_commands.json` should only be searched for when adding the `--run_output` flag, which I didn't.

Can I safely ignore this error? How can it be fixed?

**Comment by deadmoose** [comment]
> `.bazel_fix_commands.json` should only be searched for when adding the `--run_output` flag, which I didn't.

https://github.com/bazelbuild/bazel-watcher/pull/349 flipped `--run_output` on by default, so it'll always try to read that & carp that it's missing.

**Comment by achew22** [comment]
Sorry to be slow on this, I switched jobs so things are a bit hectic in my life right now. Yes that is safe to ignore. The other option would be to create either an empty `.bzel_fix_commands.json` or put something in there to invoke a tool during build errors.

Here is a simple one that invokes `bazel run //:gazelle` when it is necessary.

```
[{
    "regex": "^Check that imports in Go sources match importpath attributes in deps.$",
    "command": "bazel",
    "args":    ["run", "//:gazelle"]
}]
```

**Comment by deadmoose** [comment]
Just creating an empty one isn't enough either, since that results in a DIFFERENT error message when it fails to parse it into the expected structure. (And then continue along with the default built-in, which I'd argue is the wrong behavior when faced with explicitly malformed input)

**Comment by achew22** [comment]
Sorry, I meant empty to be an empty array, `[]` which should silence the array. English remains difficult. If you would like to enhance the error handling there, I would be very happy to merge anything in that vein.

**Comment by flolu** [comment]
@achew22 I've just added a `.bzel_fix_commands.json` with this content:
```json
[]
```
into the root of my Bazel project, but the error is still there :thinking:

**Comment by flolu** [comment]
@achew22 Thanks for your help. Adding a `.bzel_fix_commands.json` file with this content:
```json
[]
```
into the root of my Bazel project fixed the issue :+1:

**Comment by achew22** [comment]
We've solved the obvious manifestation of this issue, but I don't think we've solved the real issue. It shouldn't complain like that if there isn't a `.bazel_fix_commands.json` file. It should link to some documentation that says something like "Hey, there is a nifty feature you can turn on over here if you want to run it". This issue now tracks that usability bug.

**Comment by achew22** [comment]
This is now fixed and released in [`v0.13.0`](https://github.com/bazelbuild/bazel-watcher/releases/tag/v0.13.0). The usability issue has been addressed by printing out a banner warning each time ibazel is launched (but not on every file change) to notify of the functionality and a pointer to documentation describing how to address.
